### PR TITLE
Remove ResourceCompilerPass

### LINKED ISSUE #136: Replace ResourceCompilerPass with Flex recipe
https://github.com/FriendsOfSymfony/FOSCKEditorBundle/blob/43e0ec349cd809959f490d23e3e37b87c1314191/src/DependencyInjection/Compiler/ResourceCompilerPass.php#L28-L35

Twig is now mandatory so that `if` makes no sense. I do not like modifing parameters like this, before we didn't have recipes and this would remove 1 extra step, but now there is no need for this.

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | closes #136  
| License       | MIT

If I agree to go forward with #136. I will also provide a deprecation PR on `1.x`.

### CODE REVIEW

**Comment by kunicmarko20** [comment]
Also, I would add flex recipe for just `2.0.0`

File: src/FOSCKEditorBundle.php [hunk 1]
```diff
@@ -12,25 +12,14 @@
...
 use FOS\CKEditorBundle\DependencyInjection\FOSCKEditorExtension;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 /**
  * @author GeLo <geloen.eric@gmail.com>
  */
 class FOSCKEditorBundle extends Bundle
 {
-    /**
```
Comment (ElectricMaxxx) [inline src/FOSCKEditorBundle.php]: Do we have a note, means an issue, to add this to the reciepe later? Just not to forget it.

Comment (kunicmarko20) [reply src/FOSCKEditorBundle.php]: https://github.com/FriendsOfSymfony/FOSCKEditorBundle/issues/142

**Comment by kunicmarko20** [comment]
done :smile:

**Pull request merged**